paymentsheet/src/main/java/com/stripe/android/customersheet/CustomerSheetViewModel.kt

```diff
@@ -209,9 +210,13 @@ internal class CustomerSheetViewModel @Inject constructor(
         }
     }
 
-    @Suppress("UNUSED_PARAMETER")
     private fun onFormValuesChanged(formFieldValues: FormFieldValues?) {
-        // TODO (jameswoo) handle onFormValuesChanged
+        val isComplete = formFieldValues?.fieldValuePairs?.all {
```

Comment:
I’m a little confused as to why we need this method and associated `ViewAction`. Can we not compute the same information in a property on the view state?

Also, I’m not sure how it behaves in CustomerSheet, but in PaymentSheet, we only emit non-null `FormFieldValues` if every field is complete. Do we need the `all { it.isComplete }` check here?